### PR TITLE
Commands: Update test262 status icons

### PR DESCRIPTION
**Pull request by networkException**
This replaces the previous status icons with a set of emojis
chosen and partly created by members of the discord server :^)

### CODE REVIEW

File: src/commands/test262Command.ts [hunk 1]
```diff
@@ -73,19 +82,25 @@ export class Test262Command implements Command {
...
-        ["passed", "✅"],
-        ["failed", "❌"],
-        ["skipped", "⚠️"],
-        ["metadata_error", "📄"],
-        ["harness_error", "⚙️"],
-        ["timeout_error", "💀"],
-        ["process_error", "💥️"],
-        ["runner_exception", "🐍"],
+        ["passed", getPoggie],
+        ["failed", "🦬"],
```
Comment (bgianfo) [inline src/commands/test262Command.ts:97]: It would make the code a lot simpler if we just created functions for these?

**Review by bgianfo — CHANGES_REQUESTED** [review]

File: src/commands/test262Command.ts (hunk 1 above)
Comment (networkException) [reply src/commands/test262Command.ts:97]: Could you give an example? I'm not sure what you mean

Comment (bgianfo) [reply src/commands/test262Command.ts:97]: Even for the emoji's we don't need to fetch, if we also made them return the value through a function... the code would be alot simpler I think (maybe this is too much of a nitpick).

So we could just have it be `new Map<string, typeof getPoggie>(` and avoid the conditional call.

Comment (networkException) [reply src/commands/test262Command.ts:97]: Ah I see, sure (GitHub didn't update the converation, I just now saw the answer in my mails)

Comment (networkException) [reply src/commands/test262Command.ts:97]: Although the function signature of `getPoggie` (`(clientOrParent: Client | { client: Client }) => Promise<Emoji | null>`) would not work for just returning a unicode emoji in a string and the `Emoji` class is not really ment to be constructed manually, meaning that there has to be some checking of the return / label type anyways.

To me making a getter for just a unicode string work sounds more complicated than just keeping it this way.

Comment (bgianfo) [reply src/commands/test262Command.ts:97]: :man_shrugging: ok not going to die on the hill.

**Pull request merged**